.gitlab-ci.yml

```diff
@@ -31,32 +31,35 @@ spack_setup:
 .spack_intel:
   variables:
     SPACK_PACKAGE_COMPILER: intel
+.spack_nvhpc:
+  variables:
+    # We have to run GPU builds on GPU nodes for driver/makelocalrc reasons.
+    bb5_constraint: volta
+    SPACK_PACKAGE_COMPILER: nvhpc
 .spack_neuron:
   variables:
     SPACK_PACKAGE: neuron
-    SPACK_PACKAGE_REF: ''
+    SPACK_PACKAGE_REF: 'branch="olupton/adjust-rxd-test-tolerance"'
```

Comment:
once this is changed (after merging https://github.com/neuronsimulator/nrn/pull/1137),  we should be good

Reply:
Done now. See also https://github.com/BlueBrain/CoreNeuron/issues/505 for not having to do this dance in future.